cmd/agent/Dockerfile

```diff
@@ -22,6 +22,10 @@ RUN apt-get update && apt-get install -t bullseye-backports -qy libsystemd-dev &
   apt-get install -qy tzdata ca-certificates && \
   rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+# Add support for bcc bindings required to run the eBPF integration
+RUN echo "deb [trusted=yes] https://repo.iovisor.org/apt/bionic bionic-nightly main" | tee /etc/apt/sources.list.d/iovisor.list
```

Comment:
Suggest running this _before_ the previous `apt-get install` and adding the pkg to the above line.

Currently you're missing the `rm -rf ...` which is above this, so a small amount of cruft will be introduced into the image.

Reply:
Alternatively you could `COPY` from the build layer